packaging/deb/erigon/DEBIAN/postinst

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# This is a postinstallation script so the service can be configured and started when requested
+#
+sudo adduser --disabled-password --disabled-login --shell /usr/sbin/nologin --quiet --system --no-create-home --home /nonexistent erigon
```

Comment:
postinst is always running as root, sudo might cause issues here and it's not needed.

Reply:
Good callout, new version will address this

Reply:
Updated